scripts/dts/extract/default.py

```diff
@@ -64,6 +64,13 @@ def extract(self, node_path, prop, prop_type, def_label):
 
         insert_defs(node_path, prop_def, prop_alias)
 
+        # Generate together with string parameters like SPEED="high_speed" also
+        # definition SPEED_HIGH_SPEED=1 which can be used in preprocessor logic
+        # where we do not have strcmp()
+        if prop_type == 'string':
+            prop_val = label + '_' + str_to_label(prop_def[label]).strip('"')
```

Comment:
Could maybe make it `prop_val = label + '_' + str_to_label(prop_def[label].strip('"'))`, or have `str_to_label()` strip `"`.